kimchi/src/circuits/polynomials/generic.rs

```diff
@@ -314,8 +313,11 @@ pub mod testing {
         }
     }
 
-    impl<F: PrimeField, G: KimchiCurve<ScalarField = F>, OpeningProof: OpenProof<G>>
-        ProverIndex<G, OpeningProof>
+    impl<const ROUNDS: usize, F, G, Srs> ProverIndex<ROUNDS, G, Srs>
+    where
+        F: PrimeField,
+        G: KimchiCurve<ROUNDS, ScalarField = F>,
+        Srs: poly_commitment::SRS<G>,
```

Comment:
As I commented above, this never needed to be generic over `OpenProof`, it was previously over-constrained.